install.sh

```diff
@@ -117,7 +118,7 @@ do_setup() {
   setup_tools
   setup_vim $HVN_DEST
 
-  if test -z "$BASIC_ONLY"
+  if "$FULL_INSTALL"
```

Comment:
Ah that's interesting, thanks for the feedback.  Have made that change and updated the one in setup.hs as well.  Let me know if there's something else.